codestyle/README.md

```diff
@@ -0,0 +1 @@
+# emf.cloud codestyle
```

Comment:
Please elaborate on that README: What is the purpose of this folder and it's subfolders?